tests/FINDOLOGIC/Export/Tests/CSVSerializationTest.php

```diff
@@ -390,14 +389,12 @@ public function testAddingRelativeUrlIsNotCausingAnException(): void
     }
 
     /**
-     * @noinspection PhpMethodMayBeStaticInspection
-     *
      * Provides a dataset for testing if tab and new line characters which are removed by the sanitize method
      * don't break the CSV export.
      *
      * @return array Scenarios with value, the element class and the elements setter method name.
      */
-    public function csvSanitizedElementsInputProvider(): array
+    public static function csvSanitizedElementsInputProvider(): array
```

Comment:
:memo:  I didn't know data providers could be static. Definitely makes sense, because they are executed before the test run lifecycle starts anyway.

Reply:
With new PHP Unit versions, you even get a warning for non-static providers